test/integration/order.behavior.t.sol

```diff
@@ -76,7 +78,8 @@ contract OrderBehaviorTest is Test, ConsolidatedEvents {
             futuresMarketManager, 
             address(systemStatus), 
             GELATO, 
-            OPS
+            OPS,
+            address(0)
```

Comment:
**Question:** Why is `address(settings)` not passed in here?

Reply:
Updated. Not intentional